inc/cl-helpers.php

```diff
@@ -121,3 +121,124 @@ function uri_cl_get_vimeo_thumbnail( $src ) {
 	return explode( '_', $data->thumbnail_url )[0];
 
 }
+
+
+/**
+ * Return the hash of a variable.
+ *
+ * @param mix $var the variable to hash.
+ * @return str
+ */
+function uri_cl_get_hash( $var ) {
+	return md5( is_array( $var ) ? implode( '', $var ) : $var );
+}
+
+
+/**
+ * Return time difference in seconds.
+ *
+ * @param str $date the data string
+ * @return num
+ */
+function uri_cl_get_time_difference( $date ) {
+	$deadline = strtotime( $date );
+	$today = time();
+	return $deadline - $today;
+}
+
+
+/**
+ * Return an array with seconds, days, hours, minutes defined.
+ *
+ * @param num $secs the number of seconds togo.
+ * @return arr
+ */
+function uri_cl_expand_seconds( $secs ) {
+
+	$seconds = (int) $secs;
+	$days = floor( $seconds / 86400 );
+	$hours = floor( ( $seconds - ( $days * 86400 ) ) / 3600 );
+	$minutes = floor( ( $seconds - $days * 86400 - $hours * 3600 ) / 60 );
+	$seconds = floor( $seconds - ( $days * 86400 ) - ( $hours * 3600 ) - ( $minutes * 60 ) );
+
+	return array(
+		'seconds' => $seconds,
+		'minutes' => $minutes,
+		'hours' => $hours,
+		'days' => $days,
+		'tense' => ( $secs >= 0 ) ? 'future' : 'past',
+	);
+
+}
+
+/**
+ * Get a plain language sentence that announces the time remaining.
+ *
+ * @param arr $a an array with seconds, days, hours, minutes defined.
+ * @param arr $attributes are the shortcode attributes.
+ * @return str
+ */
+function uri_cl_get_time_left_in_words( $a, $attributes ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The function uri_cl_get_time_left_in_words() has a Cyclomatic Complexity of 19. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/app/uriweb/uri-component-library/pullRequest?prid=4536390)